packages/blockchain-info-components/index.d.ts

```diff
@@ -1,30 +1,138 @@
-import { StatelessComponent, CSSProperties } from 'react'
+import { StatelessComponent, PureComponent, CSSProperties } from 'react'
 import { DefaultTheme } from 'styled-components'
+import { IcoMoonType } from './src/Icons/Icomoon'
+// import { ImageType } from './src/Images/Images'
+
+export const BlockchainLoader: StatelessComponent<{
+  width?: string
+  height?: string
+}>
+export const Button: StatelessComponent<{
+  nature?:
+    | 'copy'
+    | 'dark'
+    | 'empty-secondary'
+    | 'empty'
+    | 'gray-3'
+    | 'green'
+    | 'light'
+    | 'primary'
+    | 'purple'
+    | 'received'
+    | 'secondary'
+    | 'sent'
+    | 'success'
+    | 'transferred'
+    | 'warning'
+    | 'white-transparent'
+  fullwidth?: boolean
+  disabled?: boolean
+  rounded?: boolean
+  bold?: boolean
+  small?: boolean
+  uppercase?: boolean
+  capitalize?: boolean
+  width?: string
+  padding?: string
+  margin?: string
+  jumbo?: boolean
+  height?: string
+  className?: string
+  onClick?: () => void
+  style?: object
+}>
 
-export function BlockchainLoader(...args: any): any
-export function Button(...args: any): any
 export function ComponentDropdown(...args): any
```

Comment:
Left this as is, since it's not a stateless component. I tried PureComponent, but was stuck on `selectedComponent`. The prop takes a component, i.e. selectedComponent{<Link />}